pkg/crc/preflight/preflight_linux.go

```diff
@@ -260,6 +257,7 @@ func getNetworkChecks(networkMode network.Mode, systemdResolved bool) []Check {
 func getPreflightChecksForDistro(distro *linux.OsRelease, networkMode network.Mode, systemdResolved bool) []Check {
 	var checks []Check
 	checks = append(checks, nonWinPreflightChecks[:]...)
+	checks = append(checks, wsl2PreflightChecks)
```

Comment:
This is confusing in my opinion... This is not a 'distro' check.
The naming of the preflights is very messy...

and this is not adjusted in the `GetAllPreflightChecks`?

Reply:
I do appreciate the attempt there ;-)